tests/snappi_tests/multidut/ecn/test_multidut_red_accuracy_with_snappi.py

```diff
@@ -90,9 +90,9 @@ def test_red_accuracy(request,
 
     snappi_extra_params.packet_capture_type = packet_capture.IP_CAPTURE
     snappi_extra_params.is_snappi_ingress_port_cap = True
-    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 2000000, 'pmax': 5}
+    snappi_extra_params.ecn_params = {'kmin': 500000, 'kmax': 900000, 'pmax': 5}
```

Comment:
pls create a seperate PR for this change, not related iteration.

Reply:
@selldinesh :please take care of this comment so we can approve/merge this asap

Reply:
@abdosi I have removed the bgp part and modified the title